src/taco-utils/helpCommandBase.ts

```diff
@@ -257,11 +264,10 @@ module TacoUtility {
          */
         private commandExists(command: string): boolean {
             for (var i in this.commandsFactory.listings) {
-                if (i === command) {
+                if (i === command || (this.commandsFactory.listings[i].aliases && (i === this.commandsFactory.listings[i].aliases[command]))) {
```

Comment:
This surprised me; I thought that command-level aliasing (template ~ templates, platform ~ platforms) was done based on the "aliases" entry of commands.json, but this does it by putting both command and subcommand aliases together in each command.

It works how it is at the moment, but I prefer the idea of having a single place where command-level aliases go.